packages/lab/src/overlay/OverlayHeader.tsx

```diff
@@ -0,0 +1,74 @@
+import { H4, Text, makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+import {
+  type ComponentPropsWithoutRef,
+  type ReactNode,
+  forwardRef,
+} from "react";
+import overlayHeaderCss from "./OverlayHeader.css";
+
+const withBaseName = makePrefixer("saltOverlayHeader");
+
+export interface OverlayPanelContentProps
+  extends ComponentPropsWithoutRef<"div"> {
+  /**
+   * Description text is displayed just below the header
+   **/
+  description?: ReactNode;
+  /**
+   * Actions to be displayed in header
+   */
+  actions?: ReactNode;
```

Comment:
This is perhaps start of new naming convention not seen on other components? If so, needing to document when/why around this. e.g. if this is design driven, might need specific positioning relationship vs prop name. 

Can anything apart from action buttons be used here? e.g. badge? icon with tooltip? etc. it may impact the naming.